Unused parameter warnings

```
In file included from src/game/world.cpp:25:
ext/seed11/include\seed11/seed11.hpp:39:45: warning: unused parameter 'b' [-Wunused-parameter]
                        acquire_seed_size_seed_seq(InputIterator b, InputIterator e) :
                                                                 ^
ext/seed11/include\seed11/seed11.hpp:39:62: warning: unused parameter 'e' [-Wunused-parameter]
                        acquire_seed_size_seed_seq(InputIterator b, InputIterator e) :
                                                                                  ^
ext/seed11/include\seed11/seed11.hpp:46:56: warning: unused parameter 'in' [-Wunused-parameter]
                        acquire_seed_size_seed_seq(std::initializer_list<V> in) :
                                                                            ^
ext/seed11/include\seed11/seed11.hpp:65:30: warning: unused parameter 'out' [-Wunused-parameter]
                        void param(OutputIterator out)
                                                  ^
```
To reproduce:
```cpp
static auto rand = seed11::make_seeded<std::mt19937>();
throw rand;
```
plzzz fix

--- a/include/seed11/seed11.hpp
+++ b/include/seed11/seed11.hpp
@@ -36,14 +36,14 @@ namespace seed11
 			}
 
 			template<typename InputIterator>
-			acquire_seed_size_seed_seq(InputIterator b, InputIterator e) :
+			acquire_seed_size_seed_seq(InputIterator, InputIterator) :
 				s(nullptr)
 			{
 
 			}
 
 			template<typename V>
-			acquire_seed_size_seed_seq(std::initializer_list<V> in) :
+			acquire_seed_size_seed_seq(std::initializer_list<V>) :
 				s(nullptr)
 			{
 
@@ -62,9 +62,9 @@ namespace seed11
 			}
 
 			template<typename OutputIterator>
-			void param(OutputIterator out)
+			void param(OutputIterator)
 			{
-				
+
 			}
 		};
 

